app/src/test/java/ar/scacchipa/twittercloneapp/viewmodel/AuthWebDialogViewModelTest.kt

```diff
@@ -49,56 +49,53 @@ class AuthWebDialogViewModelTest {
                 "scope=tweet.read%20tweet.write" +
                 "state=state&code_challenge=challenge&code_challenge_method=plain")
 
-        subject.controlRequest(uri)
-        Assert.assertEquals(subject.userAccessToken.value, null)
+
+        Assert.assertEquals(subject.tokenResource.value, null)
```

Comment:
I think you can delete this test.

Reply:
It will be removed for the next PR (about test).